samtools.wdl

```diff
@@ -452,6 +452,46 @@ task Merge {
     }
 }
 
+task Quickcheck {
+    input {
+        File inputBam
+
+        Int threads = 1
```

Comment:
Since there are no user configurable threads this can be set directly in the runtime section. (For this application more threads do not make sense anyway).